src/Juvix/Prelude/Base.hs

```diff
@@ -47,3 +47,16 @@ findRepeated = map (head . snd . fst) . findRepeatedOn id
 
 allDifferent :: forall a. (Ord a) => [a] -> Bool
 allDifferent = null . findRepeated
+
+-- Function to group elements in fixed-size chunks
+groupEvery :: Int -> [a] -> [[a]]
+groupEvery _ [] = []
+groupEvery n xs = take n xs : groupEvery n (drop n xs)
```

Comment:
use [splitAt](https://hackage.haskell.org/package/base-4.21.0.0/docs/Prelude.html#v:splitAt)

Reply:
did you push the changes?